pkg/asset/installconfig/azure/session.go

```diff
@@ -117,18 +133,47 @@ func getCredentials(fs auth.FileSettings) (*Credentials, error) {
 	if tenantID == "" {
 		return nil, errors.New("could not retrieve tenantId from auth file")
 	}
+	activeDirectoryEndpoint := fs.Values[auth.ActiveDirectoryEndpoint]
+	if activeDirectoryEndpoint == "" {
+		return nil, errors.New("could not retrieve active directory endpoint from auth file")
+	}
+
+	resourceManagerEndpoint := fs.Values[auth.ResourceManagerEndpoint]
+	if resourceManagerEndpoint == "" {
+		return nil, errors.New("could not retrieve Resource Manager Endpoint from auth file")
+	}
+
+	
+
 	return &Credentials{
 		SubscriptionID: subscriptionID,
-		ClientID:       clientID,
-		ClientSecret:   clientSecret,
-		TenantID:       tenantID,
+		ClientID:       			clientID,
+		ClientSecret:   			clientSecret,
+		TenantID:       			tenantID,
+		ActiveDirectoryEndpoint: 	activeDirectoryEndpoint,
+		ResourceManagerEndpoint: 	resourceManagerEndpoint,
 	}, nil
 }
 
 func askForCredentials() (*Credentials, error) {
-	var subscriptionID, tenantID, clientID, clientSecret string
+	var subscriptionID, tenantID, clientID, clientSecret, environmentName string
+
+	environmentOptions := []string{"public","usgovernment"}
 
 	err := survey.Ask([]*survey.Question{
+		{
```

Comment:
public should be the default. It would be nice if this prompt was only displayed if we know that this will be a usgovernment install.

Reply:
I'm not sure how we would know a US Government install without prompting them if it was or not. I'll adjust the default to be public.